docs/installation/launching-ovn-kubernetes-on-kind.md

```diff
@@ -18,15 +18,16 @@ KIND (Kubernetes in Docker) deployment of OVN kubernetes is a fast and easy mean
 - jq
 - openssl
 - openvswitch
+- Go 1.23.0 or above
```

Comment:
I took the version from the go mods here, if you think this should be different please let me know.